packages/vscode-extension/src/commands/BaseAction.ts

```diff
@@ -44,13 +43,13 @@ export default abstract class BaseAction {
 		}
 	}
 
-	protected async runSuiteCloudCommand(args: { [key: string]: string } = {} ) {
-		const suiteCloudRunnerRunResult = await new SuiteCloudRunner(this.executionPath).run({
+	protected async runSuiteCloudCommand(args: { [key: string]: string } = {}) {
+		const suiteCloudRunnerRunResult = await new SuiteCloudRunner(this.vsConsoleLogger, this.executionPath).run({
```

Comment:
Passing now the vsConsoleLogger to the SuiteCloudRunner to avoid the duplication of the "execution details" in line 52.